dmutils/content_loader.py

```diff
@@ -319,6 +318,10 @@ def get_question_by_slug(self, question_slug):
             if question.get('slug') == question_slug:
                 return question
 
+    def inject_brief_questions_into_boolean_list_question(self, brief):
```

Comment:
Not sure if this name should be more generic - we could in theory have boolean lists that aren't to do with briefs?
e.g. `inject_list_of_questions_into_boolean_list` or similar.